frappe/tests/test_query.py

```diff
@@ -201,3 +201,16 @@ def test_implicit_join_query(self):
 				fields=["name", "`tabNote Seen By`.`user` as seen_by"],
 			),
 		)
+
```

Comment:
are there any tests for nested set support in qb?

Reply:
no I'll add it in this pr

Reply:
done 👍🏿